debezium-connector-mongodb/src/main/java/io/debezium/connector/mongodb/MongoDbConnector.java

```diff
@@ -100,6 +101,8 @@ public String version() {
     @Override
     public void start(Map<String, String> props) {
         // Validate the configuration ...
+        CommonConnectorConfig.validateConnectConfig(props);
```

Comment:
How about extracting a base class for all the connectors and doing this once in its `start()` method?

Reply:
@gunnarmorling done